mantle/kola/tests/rhcos/upgrade.go

```diff
@@ -358,8 +359,23 @@ func downloadLatestReleasedRHCOS(target string) (string, error) {
 		return
 	}(releaseIndex, unique)
 
+	// In cases where there is a blocked upgrade to a new Y-stream there can be
+	// two nodes that don't have an edge to upgrade to. This is generally the
+	// latest 4.Y-1.Z and the latest 4.Y.Z. Choose the latest 4.Y.Z
+	latest := difference[0]
```

Comment:
Not new, but I think before this line we should do a length check and give a nicer error message if somehow it's empty.